ui/packages/atlasmap-core/src/services/field-action.service.ts

```diff
@@ -49,13 +48,10 @@ export class FieldActionService {
   constructor(private api: typeof ky) {}
 
   async fetchFieldActions(): Promise<boolean> {
-    return new Promise<boolean>((resolve, reject) => {
+    return new Promise<boolean>((resolve) => {
       if (this.cfg.preloadedFieldActionMetadata) {
         this.clearActionDefinitions();
-        if (
-          this.cfg.preloadedFieldActionMetadata &&
-          this.cfg.preloadedFieldActionMetadata.ActionDetails
-        ) {
+        if (this.cfg.preloadedFieldActionMetadata.ActionDetails) {
```

Comment:
Does this want to be 'if (this.cfg.preloadedFieldActionMetadata?.ActionDetails) {'

Reply:
It is already checked at L52 I think?

Reply:
yup - missed that